### ISSUE TITLE
Add Adobe

### ISSUE THREAD

**Issue by mima-mima** [robchahin/sso-wall-of-shame#325]
Adobe Creative Cloud All Apps licenses cost $259.95 per faculty/staff.
For Single Sign On , we need purchase Enterprise Named User licenses at an annual cost of $327.95 per faculty/staff.

**Comment by 7nine** [comment]
Just going to second this. I'd also like to add that Adobe can rot in hell.

**Comment by alexlushsevernunival** [comment]
Agreed. In order to get SSO you need to upgrade from a Teams package to Enterprise. If I get correct pricing for both I will add a PR.

**Comment by Anaphylaxis** [comment]
Adobe: "For 18 Acrobat Pro licenses, EACH Acrobat Pro license will cost you USD 319.08/year + taxes."
"You are currently paying USD 287.88/year + taxes per license."

**Comment by sdepablos** [comment]
We found that a lot of our employees needing Adobe Acrobat Pro were actually just fine with the ilovepdf.com webpage, at USD 0/year
